code/modules/vehicles/wheelchair.dm

```diff
@@ -108,8 +110,42 @@
 		return TRUE
 	return FALSE
 
-/obj/vehicle/ridden/wheelchair/the_whip/driver_move(mob/living/user, direction)
-	if(istype(user))
-		var/datum/component/riding/D = GetComponent(/datum/component/riding)
-		D.vehicle_move_delay = round(CONFIG_GET(number/movedelay/run_delay) * 6.7) / max(user.usable_hands, 1)
-	return ..()
+/obj/item/wheelchair
+	name = "wheelchair"
+	desc = "A collapsed wheelchair that can be carried around."
+	icon = 'icons/obj/vehicles.dmi'
+	icon_state = "wheelchair_folded"
+	base_icon_state = "wheelchair_folded"
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
+	w_class = WEIGHT_CLASS_BULKY
+	force = 8 //Force is same as a chair
+	var/unfolded_type = /obj/vehicle/ridden/wheelchair
+
+/obj/vehicle/ridden/wheelchair/MouseDrop(over_object, src_location, over_location)  //Lets you collapse wheelchair
+	. = ..()
+	if(over_object != usr || !Adjacent(usr) || !foldabletype)
```

Comment:
`if(over_object != usr || !Adjacent(usr) || !foldabletype || !ishuman(usr) || has_buckled_mobs())`
Unless I'm missing something as to why you didn't just have these combined before

Reply:
yeah fix that